src/pages/[slug].tsx

```diff
@@ -7,16 +7,21 @@ import LoadConfigComponent from '../components/Config/LoadConfig';
 import { Config } from '../tools/types';
 import { useConfig } from '../tools/state';
 import Layout from '../components/layout/Layout';
+import { getConfig } from '../tools/getConfig';
+import { dashboardNamespaces, loginNamespaces } from '../tools/translation-namespaces';
 
 export async function getServerSideProps(
-  context: GetServerSidePropsContext
+  context: GetServerSidePropsContext,
+  res: Response,
+  req: Request,
+  locale: string
 ): Promise<{ props: { config: Config } }> {
   const configByUrl = context.query.slug;
   const configPath = path.join(process.cwd(), 'data/configs', `${configByUrl}.json`);
   const configExists = fs.existsSync(configPath);
   if (!configExists) {
     // Redirect to 404
-    context.res.writeHead(301, { Location: '/404' });
+    // context.res.writeHead(301, { Location: '/404' });
```

Comment:
1. Is this why we receive a 404 in the console? When I commented this out, I don't get it anymore
2. Can't we use here the router instead of the response?